raiden/tests/scenarios/ci/Scenario-Stress-Hub.yaml

```diff
@@ -0,0 +1,113 @@
+            - transfer: {from: 3, to: 8, amount: 1, expected_http_status: "(200|408)"}
+            - transfer: {from: 3, to: 4, amount: 1, expected_http_status: "(200|408)"}
+            - transfer: {from: 3, to: 2, amount: 1, expected_http_status: "(200|408)"}
+            - transfer: {from: 3, to: 9, amount: 1, expected_http_status: "(200|408)"}
+            - transfer: {from: 4, to: 7, amount: 1, expected_http_status: "(200|408)"}
+            - transfer: {from: 4, to: 9, amount: 1, expected_http_status: "(200|408)"}
+            - transfer: {from: 4, to: 6, amount: 1, expected_http_status: "(200|408)"}
+            - transfer: {from: 4, to: 3, amount: 1, expected_http_status: "(200|408)"}
+            - transfer: {from: 4, to: 8, amount: 1, expected_http_status: "(200|408)"}
+            - transfer: {from: 5, to: 4, amount: 1, expected_http_status: "(200|408)"}
+            - transfer: {from: 5, to: 2, amount: 1, expected_http_status: "(200|408)"}
+            - transfer: {from: 5, to: 1, amount: 1, expected_http_status: "(200|408)"}
+            - transfer: {from: 5, to: 3, amount: 1, expected_http_status: "(200|408)"}
+            - transfer: {from: 5, to: 8, amount: 1, expected_http_status: "(200|408)"}
+            - transfer: {from: 6, to: 4, amount: 1, expected_http_status: "(200|408)"}
+            - transfer: {from: 6, to: 7, amount: 1, expected_http_status: "(200|408)"}
+            - transfer: {from: 6, to: 8, amount: 1, expected_http_status: "(200|408)"}
+            - transfer: {from: 6, to: 9, amount: 1, expected_http_status: "(200|408)"}
+            - transfer: {from: 6, to: 3, amount: 1, expected_http_status: "(200|408)"}
+            - transfer: {from: 7, to: 5, amount: 1, expected_http_status: "(200|408)"}
+            - transfer: {from: 7, to: 4, amount: 1, expected_http_status: "(200|408)"}
+            - transfer: {from: 7, to: 8, amount: 1, expected_http_status: "(200|408)"}
+            - transfer: {from: 7, to: 6, amount: 1, expected_http_status: "(200|408)"}
+            - transfer: {from: 7, to: 2, amount: 1, expected_http_status: "(200|408)"}
+            - transfer: {from: 8, to: 9, amount: 1, expected_http_status: "(200|408)"}
+            - transfer: {from: 8, to: 5, amount: 1, expected_http_status: "(200|408)"}
+            - transfer: {from: 8, to: 4, amount: 1, expected_http_status: "(200|408)"}
+            - transfer: {from: 8, to: 3, amount: 1, expected_http_status: "(200|408)"}
+            - transfer: {from: 8, to: 6, amount: 1, expected_http_status: "(200|408)"}
+            - transfer: {from: 9, to: 3, amount: 1, expected_http_status: "(200|408)"}
+            - transfer: {from: 9, to: 4, amount: 1, expected_http_status: "(200|408)"}
+            - transfer: {from: 9, to: 2, amount: 1, expected_http_status: "(200|408)"}
+            - transfer: {from: 9, to: 7, amount: 1, expected_http_status: "(200|408)"}
+            - transfer: {from: 9, to: 8, amount: 1, expected_http_status: "(200|408)"}
```

Comment:
IMO this is incomplete without asserting for the state of the channels at the end.

Reply:
Thanks Cosmin, here again, we can not check for channel states, since we expect as a response per payment a 200 or 408. Only in the first case one token would be added to the channel balance. We could test here whether the network is stable while we stress it. Do you think this makes sense?

Reply:
Thanks Cosmin, here again, we can not check for channel states, since we expect as a response per payment a 200 or 408. Only in the first case one token would be added to the channel balance. We could test here whether the network is stable while we stress it. Do you think this makes sense?